pipeline/config/elastx/sc-config/cluster.tfvars

```diff
@@ -0,0 +1,150 @@
+# your Kubernetes cluster name here
+cluster_name = "pipeline-elastx-sc"
+
+# SSH key to use for access to nodes
+public_key_path = "/github/workspace/apps/pipeline/config/elastx/id_rsa.pub"
+
+# image to use for bastion, masters, standalone etcd instances, and nodes
+# you can find the available images with `openstack image list`
+# image = "ubuntu-24.04-server-latest"
+image_uuid = "3a341474-6c80-42df-aafb-bb591fc9d96a"
+
+# user on the node (ex. core on Container Linux, ubuntu on Ubuntu, etc.)
+ssh_user = "ubuntu"
+
+# 0|1 bastion nodes
+number_of_bastions = 0
+
+# standalone etcds
+number_of_etcd = 0
+
+# control plane availability zones
+az_list = [
+  "sto1",
+  "sto2",
+  "sto3"
+]
+
+# masters
+number_of_k8s_masters = 0
+
+number_of_k8s_masters_no_etcd = 0
+
+number_of_k8s_masters_no_floating_ip = 0
+
+number_of_k8s_masters_no_floating_ip_no_etcd = 0
+
+# Flavor depends on your openstack installation
+# you can get available flavor IDs through `openstack flavor list`
+k8s_masters = {
+  "control-plane-1" = {
+    "az"          = "sto1"
+    "flavor"      = "4e4318bb-8cd9-4274-8150-4eaeb478b8e5" #2C-4GB-60GB
```

Comment:
Please change all to equivalent v2 flavours, that might help your issue with control plane nodes joining, given that they should be more performant.